index.js

```diff
@@ -8,16 +8,76 @@ const validation = require('./src/validation.js');
 const library = require('./src/library.js');
 
 /**
- * Creates OS based shortcuts for files, folders, and applications.
+ * OPTIONAL: console.error is called by default if verbose: true.
+ *
+ * Your own custom logging function called with helpful warning/error
+ * messages from the internal validators. Only used if verbose: true.
+ *
+ * @callback {Function} CUSTOMLOGGER
+ * @param    {string}   message       The human readable warning/error message
+ * @param    {object}   [error]       Sometimes an error or options object is passed
+ * @param    {Array}    [asdf]        asdf
```

Comment:
Did you wanted to push these lines?

Reply:
If you are asking about the formatting (number of spaces), yes, I added a rule to my linter to include `@callback` in the formatting, and then the linter fixed the spacing.

Reply:
Ah, I see what you were referring to now. cleaned up